great-idea/css/index.css

```diff
@@ -63,4 +63,136 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.main{
+	width: 1000px;
+	margin: 0 auto;
+}
+.header{
+	display: flex;
+	justify-content: space-around;
+}
+
+.header nav{
+	display: flex;
+	justify-content: space-around;
+	flex-basis: 80%;
+
+}
+ nav a {
+	 padding: 20px;
+	 color: grey;
+ }
+
+
+ .get{
+	 border: 1px solid black;
+	 width: 120px;
+	 text-align: center;
+	 padding: 10px;
+	 margin-left: 50px;
+	 
+	 
+	
+ }
+
+ #company-name{
+	 font-size: 65px;
+	 align-items: center;
+	 margin-left: 20px;
+
+ }
+
+ 
+ .first-section{
+	 display: flex;
+	 justify-content: space-around;
+	 align-items: center;
+	 margin-top: 40px;
+
+ }
+
+ .image{
+	 margin-left: 200px;
+	 margin-right: 30px;
```

Comment:
Giving the image property of margin right of 30px causes a misalignment on the right side of the page.
Also, instead of margin left, I would have liked to see you use flexbox property of justify-content: space-between.